falco.yaml

```diff
@@ -525,7 +538,7 @@ json_include_tags_property: true
 # output mechanism. By default, buffering is disabled (false).
 buffered_outputs: false
 
-# [Experimental] `rule_matching`
+# [Incubating] `rule_matching`
```

Comment:
Experimental -> Incubating

Reply:
urgh ok here we should maybe also introduce a synonym `rules_matching` @leogr?

Reply:
I think singular is ok, in this case; it is `rule-matching algorithm`, feels ok `rule_matching: foo` to me.